packages/core/src/tooltip/Tooltip.tsx

```diff
@@ -111,7 +111,7 @@ export const Tooltip = forwardRef<HTMLDivElement, TooltipProps>(
             ) : (
               <>
                 {getIcon({})}
-                <span className={withBaseName("body")}>{title}</span>
+                <span className={withBaseName("title")}>{title}</span>
```

Comment:
Is there a reason for this name change other than preference? I think we used to call it a tooltip body but I don't have a preference in the name.